app/src/containers/FeedData.js

```diff
@@ -21,6 +21,14 @@ const updateQuery = (prev, { subscriptionData: { data }}) => {
     return {
       feed: [node, ...prev.feed]
     };
+  } else if (mutation === 'DELETED') {
+    const deletedNodeId = previousValues.id.match(/^StringIdGCValue\((.+)\)/);
```

Comment:
deleted node id is returned in `previousValues` in the following format: `StringIdGCValue(...)`, parsing with regex